roundwared/recording_collection.py

```diff
@@ -259,5 +259,8 @@ def _update_playlist_timed(self):
         # Make a list of all assets that aren't banned.
         self.playlist_timed = [item.asset for item in playlist
                                if not self._banned(item.asset)]
+        # filter playlist_timed for current tag filters by comparing
+        # to self.all which has already been filtered by tags
+        self.playlist_timed = set(self.playlist_timed).intersection(self.all)
```

Comment:
This works, but technically should be part of the list comprehension in  260-261.

Should be tested ;)